doc/man7/ossl_store.pod

```diff
@@ -44,7 +44,10 @@ other encoding is undefined.
 
 =head2 A generic call
 
- OSSL_STORE_CTX *ctx = OSSL_STORE_open("file:/foo/bar/data.pem");
+ #include <openssl/ui.h> // for UI_get_default_method
+ #include <openssl/store.h>
+
+ OSSL_STORE_CTX *ctx = OSSL_STORE_open("file:/foo/bar/data.pem", UI_get_default_method(), NULL, NULL, NULL);
```

Comment:
It could use the _ex version here instead..

Reply:
Also the line should be split as it is too long.

Reply:
> It could use the _ex version here instead..

How do you get:
`OSSL_LIB_CTX *libctx, const char *propq,`
`const OSSL_PARAM params[],`

Since I don't know I used the simpler version.